lib/http/service.js

```diff
@@ -63,6 +63,7 @@ module.exports = (container) => {
   const { builder } = require('./endpoint');
   const { emptyAuthInjector, authHandler, queryOptionsHandler, userAgentHandler } = require('./preprocessors');
   const endpoint = builder(container, [ emptyAuthInjector, authHandler, queryOptionsHandler, userAgentHandler ]);
+  const anonymousEndpoint = builder(container, [ emptyAuthInjector, queryOptionsHandler, userAgentHandler ]);
```

Comment:
I'm a little worried about someone updating the list for `endpoint` but not `anonymousEndpoint`. What do you think about code like what's below? Alternatively, a code comment would probably suffice.

```js
const preprocessors = [ emptyAuthInjector, authHandler, queryOptionsHandler, userAgentHandler ];
const endpoint = builder(container, preprocessors);
// without() from Ramda
const anonymousEndpoint = builder(container, without([authHandler], preprocessors));
```